common/scala/src/main/resources/application.conf

```diff
@@ -82,4 +82,7 @@ whisk {
         activations-ddoc = "whisks.v2"
         activations-filter-ddoc = "whisks-filters.v2"
     }
+
+    controller {
```

Comment:
Had to add this so that `loadConfigOrThrow[ControllerConfig](ConfigKeys.controller)` does not fail.

Reply:
I think this i s because you're using ansible still for the values.

Reply:
tidStrides is an optional value in the controller section here. However, the controller section has to be defined to prevent a runtime here, @rabbah.

Reply:
I think what you want here is:
```
    controller {
        tidStrides = ${?CONTROLLER_INSTANCES}
    }
```
that will default tidStrides to the value of the CONTROLLER_INSTANCES envar if CONFIG_whisk_controller_tidStrides isn't set.

Reply:
It feels a bit odd to add "controller" to the common configuration, does it? Also you don't want to use camelCased but rather kebab-cased fieldnames, otherwise the override through the env-convert script won't work.

Should we rather put this under a "transactions" key? `whisk.transactions.strides`

Reply:
To add to dave's comment, if you want to get rid of the `Option` in the code itself you can do this:

```
whisk {
  transactions {
    strides = 1
    strides = ${?CONTROLLER_INSTANCES}
  }
}
```

Reply:
> whisk.transactions.strides
+1 good suggestion.

Reply:
the `option` makes sense with the code below where it default to number of instances; fits as a nice idiom.